docs/content/docs/testing/tinylicious.md

```diff
@@ -1,17 +1,92 @@
 ---
 title: Tinylicious
 menuPosition: 1
-status: unwritten
+editor: sdeshpande3
 ---
 
 ## What is Tinylicious?
 
-Our local service used for prototyping
-
-Link to repo
+Tinylicious is a local, in-memory service used for prototyping and development purpose. You can find the code for the [Tinylicious service](https://github.com/microsoft/FluidFramework/tree/main/server/tinylicious) and [Tinylicious client](https://github.com/microsoft/FluidFramework/tree/main/packages/framework/tinylicious-client). It can be instantiated against both `AzureClient` and `TinyliciousClient` for testing purpose.
 
 ## Using Tinylicious locally
 
+You can run tinylicious locally by executing the following command,
+
+```sh
+npx tinylicious
+```
+
+By default, Tinylicious runs on port `7070`, however you can change port by navigating to `server/tinylicious` and running the below command on `Windows Powershell` by specifying the port number,
+
+```sh
+$env:PORT=6502
+npm run start
+```
+
+Now, you can navigate to `http://localhost:6502` on your browser to see Tinylicious service up and running.
+
 ## How to deploy using Tinylicious
 
+The `AzureClient` supports both instantiating against a deployed Azure Fluid Relay service instance for production scenarios, as well as against a local, in-memory service instance, known as Tinylicious, for development purposes.
+
+You can connect to a live Azure Fluid Relay instance by passing in the tenant ID as `local`, the orderer and storage URLs  pointing to the Tinylicious instance on the default values of `http://localhost:7070` to connect to a local Tinylicious server for development purposes. You can use `InsecureTokenProvider` for token resolution while running the service locally for development purposes.
+
+In the below code snippet, `AzureClient` is pointing to Tinylicious service. This is providing you with an added feature to work with both the local Tinylicious service and the deployed Azure Fluid Relay service. `TinyliciousClient` only works with local Tinylicous service.
+
+```typescript
+import { AzureFunctionTokenProvider, AzureClient, AzureConnectionConfig } from "@fluidframework/azure-client";
+import { InsecureTokenProvider } from "@fluidframework/test-client-utils";
+
+const user = {
+    id: "UserId",
+    name: "Test User",
+};
+
+const connectionConfig: AzureConnectionConfig = {
+    tenantId: "local",
+    tokenProvider: new InsecureTokenProvider("fooBar", user),
+    // if you're running Tinylicious on a non-default port, you'll need change these URLs
+    orderer: "http://localhost:7070",
+    storage: "http://localhost:7070",
+};
+
+const client = new AzureClient(connectionConfig);
```

Comment:
The `AzureClient` now takes in a prop that contains the config as a parameter. The config cannot be passed in directly to initialize it.